### DISCUSSION TITLE
Unable to connect via mintegration with VPL15 TOP

### DISCUSSION THREAD

**Discussion by Czdav1** [veista/nilan#148] [Q&A]
Hi, I have a Nilan VPL 15 - photo. Waveshare converter 485 to ETH Poe - photo, connected 485A and 485B. Unable to connect when starting integration. Don't know where to look for a problem?

![image](https://github.com/user-attachments/assets/bde24b4d-61dd-4f27-8d1d-2bebceb17c2e)

![image](https://github.com/user-attachments/assets/9d52d4cd-ccb9-457c-b873-fbfedb10aaad)

![image](https://github.com/user-attachments/assets/581d9abf-60f7-48da-a20b-0f08ac2cbc01)

![image](https://github.com/user-attachments/assets/b5134b40-5496-4478-a287-5a22f6b4608f)

![image](https://github.com/user-attachments/assets/08e707a0-d83d-42b5-b67e-01d0acf8304c)

**Comment by veista** [comment]
Your RS485 settings are different from Nilan default. You should check those. Also check that the bridge is reachable from network.

**Reply by berlinloopbrasil-hub** [reply]
hello I have the same issue but different set up. I can ping the ip and see the port but in integration it doesn't conect.
<img width="598" height="810" alt="Screenshot 2025-11-06 at 19 36 40" src="https://github.com/user-attachments/assets/06265f74-98d0-4b53-9c66-dafa404f2b37" />
<img width="570" height="784" alt="Screenshot 2025-11-06 at 19 36 28" src="https://github.com/user-attachments/assets/3daec6c3-1514-4504-8a4f-7e5ed4283a4e" />

**Reply by berlinloopbrasil-hub** [reply]
The RX Light also blinks but not the TX. 
![IMG_7224](https://github.com/user-attachments/assets/4a085543-c0c6-42d3-a2fa-32a223fb8bd0)

**Reply by veista** [reply]
You might want to check your UART settings.

**Reply by berlinloopbrasil-hub** [reply]
Here is what I got:
Mode:               Modbus TCP ⇔ Modbus RTU
Baud rate:          9600
Data bits:          8
Parity:             None
Stop bits:          1
Flow control:       Disable
485 mode:           Enable
Baudrate adaptive:  Disable

not much to go wrong i thought 
May be I missed something

**Reply by berlinloopbrasil-hub** [reply]
I also recognized that, when trying to install the modul with the IP 10.0.0.75 and Port 502 that on the device the TX LED Flashes up couple of times, so it gets some Pings from HA. The thing that worries me is the ID. If its not 1 nor 30 and no dipswitches on the inside electric board there is no way to know. Looks like it doesnt trigger a response but ... the control panel works just finde with all the data.

**Reply by veista** [reply]
The control panel uses a separate bus on CTS602.

**Comment by veista** [comment]
https://github.com/veista/nilan/wiki/RS485-TO-ETH-(B)-to-CTS602-Installation-Guide

https://github.com/veista/nilan/wiki/Wiring-for-CTS602
